src/atoms/list-item-content/list-item-content.js

```diff
@@ -1,19 +1,20 @@
 import styled from 'styled-components';
-import { space, themeGet, lineHeight } from 'styled-system';
+import { space, themeGet, lineHeight, width, flex } from 'styled-system';
 import { createSkeletonElement } from '@trainline/react-skeletor';
 import { flipOrder } from 'styles';
 import { fontSize } from 'utils';
 import { IconCircle } from 'atoms';
 
 const makeListItemTitle = styled.h3.attrs({
-	fontSize: [3, 4],
+	fontSize: [1, 2],
```

Comment:
can you pass fontSize in as an optional prop?
```
fontSize: props => props.fontSize || [1, 2]
```

Reply:
Done! Good idea.